docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
       - "DB_HOST=lime-db"
       - "DB_PASSWORD=secret"
       - "ADMIN_PASSWORD=foobar"
+      # Example: If you require an empty table prefix
```

Comment:
*If you require an empty table prefix, use a space as the DB_TABLE_PREFIX*

Reply:
Done